src/corelibs/U2Gui/src/util/CreateAnnotationWidgetController.h

```diff
@@ -149,12 +150,14 @@ private slots:
 
     CreateAnnotationModel model;
     GObjectComboBoxController *occ;
-    CreateAnnotationWidget *w;
+    QScopedPointer<CreateAnnotationWidget> w;
     SaveDocumentController *saveController;
 
     QString GROUP_NAME_AUTO;
     static const QString DESCRIPTION_QUALIFIER_KEY;
     static const QString SETTINGS_LASTDIR;
+
+    Q_DISABLE_COPY(CreateAnnotationWidgetController)
```

Comment:
OK, nice, you've forbidden the "copy" behavior, but what's about "move"? There's a good macro `Q_DISABLE_COPY_MOVE`, but it's available starting with Qt 5.13.0 and we use 5.12.9. I suggest you deleting the "move" constructor and the "move" `operator=`.

Reply:
nit: We are still in Qt 5.4 compatibility mode.

We can escalate this question on the meeting and increase the version of QT to a newer one, but historically it should be the version supported by the Debian stable, which is as far as I remember is 5.7 today.